controller/share_manager_controller.go

```diff
@@ -35,6 +36,8 @@ import (
 	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
 )
 
+const shareManagerLeaseDurationSeconds = 7 // This should be slightly more than twice the share-manager lease renewal interval.
```

Comment:
Can you explain the statement `This should be slightly more than twice the share-manager lease renewal interval.`? How to determine the value?

Reply:
I will defer this question to @james-munson

Reply:
I think I got that from a discussion of node status updates in Kubernetes, in which the stale time should allow for at least N missed updates.  Something like 
> --node-monitor-grace-period duration     Default: 40s  
```
Amount of time which we allow running Node to be unresponsive before marking it unhealthy. Must be N times more than kubelet's nodeStatusUpdateFrequency, where N means number of retries allowed for kubelet to post node status.
```
So the update interval of 3 seconds and the stale time of 7 seconds are a compromise between API activity and length of outage.